### PR TITLE
DOCS-3153 Authentication Protocols for API and Multistep API Tests Guide

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds an Authentication Protocols for API and Multistep API Tests guide + updates the Running Tests on an Application That Requires Authentication for Browser Tests guide.

### Motivation

DOCS-3153

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by estherk15 — APPROVED** [review]

File: content/en/synthetics/guide/authentication-protocols.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+[3]: /synthetics/api_tests/websocket_tests/
+{{% /tab %}}
+{{% tab "AWS Signature" %}}
+
+Click **AWS Signature**, enter an Access Key ID and Secret Access Key, and optionally, a region, service name, and session token. AWS Signature authentication is supported in [HTTP tests][1], [multistep API tests][2], and [WebSocket tests][3].
+
+[1]: /synthetics/api_tests/http_tests/
+[2]: /synthetics/multistep/
+[3]: /synthetics/api_tests/websocket_tests/
+{{% /tab %}}
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/synthetics/guide/authentication-protocols.md:68]: Hugo has recommended putting Client Certificate last specifically, not sure if we want to shift things around.

**Review by HugoPct — APPROVED** [review]

File: content/en/synthetics/guide/authentication-protocols.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (HugoPct) [reply content/en/synthetics/guide/authentication-protocols.md:68]: we sorted by popularity

**Comment by HugoPct** [comment]
couple of updates suggested, otherwise we are good to publish

File: content/en/synthetics/guide/authentication-protocols.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+[1]: /synthetics/api_tests/http_tests/
+[2]: /synthetics/multistep/
+{{% /tab %}}
+{{% tab "OAuth 2.0" %}}
+
+Click **OAuth 2.0**, select a grant type (**Client Credentials** or **Resource Password**), and include an Access Token URL, Client ID, and Client Secret. Select a token API authentication method (**Send as Basic Auth header** or **Send client credentials in body**) and optionally, include an audience, resource, and scope. OAuth 2.0 authentication is supported in [HTTP tests][1], [multistep API tests][2], and [WebSocket tests][3].
+
+[1]: /synthetics/api_tests/http_tests/
+[2]: /synthetics/multistep/
+[3]: /synthetics/api_tests/websocket_tests/
```
Comment (alai97) [inline content/en/synthetics/guide/authentication-protocols.md:51]:
```suggestion
```

File: content/en/synthetics/guide/authentication-protocols.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+{{% /tab %}}
+{{% tab "AWS Signature" %}}
+
+Click **AWS Signature**, enter an Access Key ID and Secret Access Key, and optionally, a region, service name, and session token. AWS Signature authentication is supported in [HTTP tests][1], [multistep API tests][2], and [WebSocket tests][3].
+
+[1]: /synthetics/api_tests/http_tests/
+[2]: /synthetics/multistep/
+[3]: /synthetics/api_tests/websocket_tests/
```
Comment (alai97) [inline content/en/synthetics/guide/authentication-protocols.md:67]:
```suggestion
```

File: content/en/synthetics/guide/authentication-protocols.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+{{% /tab %}}
+{{% tab "NTLM" %}}
+
+Click **NTLM**, enter a username and password, and optionally, a domain and work station. NTLM authentication is supported in [HTTP tests][1] and [multistep API tests][2], and [WebSocket tests][3].
+
+[1]: /synthetics/api_tests/http_tests/
+[2]: /synthetics/multistep/
+[3]: /synthetics/api_tests/websocket_tests/
```
Comment (alai97) [inline content/en/synthetics/guide/authentication-protocols.md:59]:
```suggestion
```

**Pull request merged**